getting_started/scripting/gdscript/gdscript_basics.rst

```diff
@@ -1231,10 +1231,10 @@ with the new value. Vice-versa, when ``variable`` is accessed, the *getter* func
     var myvar setget my_var_set, my_var_get
 
     func my_var_set(new_value):
-        my_var = new_value
+        myvar = new_value
```

Comment:
It seems to me it was supposed to be `my_var`. It's the previous declaration that is wrong. For instance, there are other references to `my_var` in the next code block.

Reply:
Agree, I would rename the declaration instead.

Reply:
good catch! lemme fix that once more 😅